meta_package_manager/bitbar/meta_package_manager.7h.py

```diff
@@ -126,30 +151,36 @@ def print_menu():
         " ⚠️{}".format(total_errors) if total_errors else ""))
 
     # Print a full detailed section for each manager.
-    echo("---")
-    for manager in managers:
+    if recursive_menu:
+      submenu="--"
+    else:
+      submenu=""
 
+    if recursive_menu:
+        echo("---")
+    for manager in managers:
+        if not recursive_menu:
+            echo("---")
 
         if manager['error']:
-          # for line in manager['error'].strip().split("\n"):
-          #     echo("{:<23} {} | color=red font=Menlo size=10 trim=false emojize=false".format(manager.name + ':', line))
-          print_error(manager['error'])
+            print_error(manager['error'], manager['name'])
```

Comment:
Missed this one. I have only seen general error for managers, not error on one package only. This one might be harder to solve. If it is a general error for a manager you probably want to replace the submenu with the error, otherwise you want the error for the specific package in the sub-menu. 

I'll see what I can come up with.